routes/user.js

```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router();
+const userCtrl = require('../controllers/user');
+
+
+// GET /user
+router.get('/user', isLoggedIn, userCtrl.index);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
router.get("/user", isLoggedIn, userCtrl.index);
```